1-js/02-first-steps/14-switch/article.md

```diff
@@ -26,71 +26,71 @@ switch(x) {
 }
 ```
 
-- The value of `x` is checked for a strict equality to the value from the first `case` (that is, `value1`) then to the second (`value2`) and so on.
-- If the equality is found, `switch` starts to execute the code starting from the corresponding `case`, until the nearest `break` (or until the end of `switch`).
-- If no case is matched then the `default` code is executed (if it exists).
+- O valor de `x` é comparado por meio de uma igualdade exata ao valor do primeiro `case` (isto é, ao `valor1`), a seguir ao do segundo (`valor2`) e assim sucessivamente.
+- Se uma igualdade for encontrada, o `switch` começa a executar o código a partir do início do `case` correspondente, até ao próximo `break` (ou até ao fim do `switch`).
+- Se nenhum `case` é equiparado então o código em `default` é executado (se existir).
 
-## An example
+## Um exemplo
 
-An example of `switch` (the executed code is highlighted):
+Um exemplo de `switch` (o código executado está em destaque):
 
 ```js run
 let a = 2 + 2;
 
 switch (a) {
   case 3:
-    alert( 'Too small' );
+    alert( 'Muito baixo' );
     break;
 *!*
   case 4:
-    alert( 'Exactly!' );
+    alert( 'Exacto!' );
     break;
 */!*
   case 5:
-    alert( 'Too big' );
+    alert( 'Muito alto' );
     break;
   default:
-    alert( "I don't know such values" );
+    alert( "Não conheço tais valores" );
 }
 ```
 
-Here the `switch` starts to compare `a` from the first `case` variant that is `3`. The match fails.
+Aqui o `switch` começa por comparar `a` à variante no primeiro `case`, que é `3`. A correspondência falha.
 
-Then `4`. That's a match, so the execution starts from `case 4` until the nearest `break`.
+Então a `4`. Existe uma correspondência, e assim a execução começa a partir do `case 4` até ao próximo `break`.
```

Comment:
**Sugestão de alteração**:

`Então a 4` ficou com uma pronúncia estranha, alterar por 

"**Na sequência, no segundo caso**"

Ou 

"**Então, no segundo caso**"

Reply:
Changed to:

> A seguir a `4`.

Is it all right?